src/components/ui-shell/_header.scss

```diff
@@ -29,6 +29,15 @@
     border: mini-units(0.5) solid transparent;
   }
 
+  .#{$prefix}--header__action > svg[data-navigation-menu-panel-collapse-icon],
```

Comment:
Do we need the `data-*` attributes in the selectors here?

Reply:
we can remove those attribute selectors if we can assume that this element will only contain the expand and collapse svg icons. would it be alright to make that assumption?

Reply:
Yeah! I think so 😄

Reply:
actually I think I misread. The data attributes are here to differentiate between which icon is hidden. Removing them would hide both icons at the same time

Reply:
We should really find a better solution over `data-*` attributes for cross-framework support. Should we just add a class to help out here? Or could we update the markup in some way to make it easier?

Reply:
Could we update the selector to something like:

```
.#{$prefix}--header__menu-trigger.#{$prefix}--header__action > svg {
  // ...
}
```

And then have a stateful selector that's applied to distinguish between open/closed?

Reply:
Looking at implementation now, do we need both icons in the markup or can we swap them in/out? If we need both of them in, should we just have a class th at is added to each to specify that their purpose is?

Reply:
should be resolved now!